applications/luci-app-usteer/Makefile.txt

```diff
@@ -0,0 +1,11 @@
+# See /LICENSE for more information.
```

Comment:
Makefiel filename is just "Makefile".
It can't be "Makefile.txt" as you now have.

Reply:
renamed